### DISCUSSION TITLE
tutorials/cko-product-workshop/02-what-is-graphql

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#81] [course feedback]
# tutorials/cko-product-workshop/02-what-is-graphql

https://www.apollographql.com/tutorials/cko-product-workshop/02-what-is-graphql

**Comment by BolajiAyodeji** [comment]
The starting point API mentioned in this lesson is down (https://comet-cruises-api.herokuapp.com). You might want to consider redeploying somewhere else. I thought you should know; thank you.
